packages/cadl-ranch-specs/http/azure/core/basic/main.tsp

```diff
@@ -20,7 +20,9 @@ using TypeSpec.Versioning;
 )
 namespace _Specs_.Azure.Core.Basic;
 
+@doc("The version of the API.")
 enum Versions {
+  @doc("The version 2022-12-01-preview.")
```

Comment:
comment here too but do we really need that doc on version enum, not sure what are people going to be commenting apart from the same thing again

Reply:
This PR is just to make the e2e test pass.

Reply:
The point of the rule is to make sure that enums and enum values have documentation. It's a slippery slope to try and decide that some enums are self-explanatory enough not to require it. If they are, then you could just suppress it.

Reply:
But would the version enum be carried over to the generated client?

Reply:
sure in general we can't, but for `Versions` we can know it is the version enum. I am saying that as this is a change all the spec will most likely need a change and I don't think is very valuable. But not gonna block on this.